CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+<a name="0.2.3"></a>
+## [0.2.3](https://github.com/Ninja-Squad/ngx-speculoos/compare/v0.2.2...v0.2.3) (2018-06-17)
+
```

Comment:
No message to say that you added a small demo app?

Reply:
Unfortunately, no. The standard-version script ignores docs changes. And the docs is abysmal, as always, so I don't know if that can be changed. Not a big deal, though.